pkgs/build-support/haxe/override_haxe_lib_from_hmm.nix

```diff
@@ -0,0 +1,103 @@
+# This tool can be used to convert as set of derivation containing haxe dependncies (as from haxe-packages.nix) to use version specified in an hmm.json file, used by certain project to lock inputs.
+# To update (or initiate it) for a package:
+# 1. copy the hmm.json file in the package folder
+# 2. run update_hmm_hashes.py (you will to have nix-prefetch-git in the PATH, which is packaged in nixpkgs under the same name).
+#    It take as argument a path to the hmm.json file, and a path to an output json file that contains the hashes for the dependancies (usually named hashes.json)
+{
+  lib,
+  haxePackages,
+  fetchgit,
+  fetchzip,
+}:
+
+{
+  hmm_attrset ? null,
+  hmm_file ? null,
+  hashes_attrset ? null,
+  hashes_file ? null,
```

Comment:
In general, nixpkgs uses kebab-case and camelCase not snake_case. No rule against it -- just looks odd to me.

Reply:
That should not a problem with the new name that doesn’t have (pseudo) space at all